test/extended/testdata/builds/build-pruning/successful-pipeline.yaml

```diff
@@ -0,0 +1,86 @@
+kind: "BuildConfig"
+apiVersion: "v1"
+metadata:
+  name: "successful-pipeline"
+spec:
+  successfulBuildsHistoryLimit: 2
+  strategy:
+    jenkinsPipelineStrategy:
+      jenkinsfile: |-
```

Comment:
yeah, here too, we could do something much simpler/shorter ... simply sleep for x amount of time, and then return

Reply:
+1, and don't run on a slave.